_data/sidebear.yml

```diff
@@ -0,0 +1,9 @@
+extensionName: Sidebear
+extensionPath: Sidebear.roboFontExt
+description: An Inspector panel for manipulating your glyph’s sidebearings.
+developer: Ryan Bugden
+developerURL: https://ryanbugden.com
+repository: https://github.com/ryanbugden/Sidebear
+infoPath: https://github.com/ryanbugden/Sidebear/blob/master/Sidebear.roboFontExt/info.plist
```

Comment:
the direct path is `https://raw.githubusercontent.com/ryanbugden/Sidebear/master/Sidebear.roboFontExt/info.plist`